docs/DEPLOYMENT.md

```diff
@@ -217,58 +233,141 @@ If on openshift verify if the ui route is accessible
 
 Open: `oc get routes ui --template='https://{{ .spec.host }} '`
 
-## Deploying for the first time?
+Note: In UI you won't be able to resources as there are not any in the db. You can see Categories on the left side.
+
+Update your GitHub OAuth created with the UI route in place of `Homepage URL` and `Authorization callback URL`.
 
-> Note: Make sure you have added your name in the scopes present [config.yaml][config.yaml]
+### Add resources in DB
 
-1. Open currently deployed hub in the browser
-2. Login into the Hub
-3. After successful login, now open the terminal and follow below steps.
-4. Exec into the database pod
+> Note: Make sure you have added your name in the scopes present [config.yaml][config.yaml]. You will need additional scope i.e. catalog-refresh to add resources.
+
+Now, follow the below steps
+
+1. Login through the UI. Click on `Login` on right corner and then `Sign in with GitHub`.
+
+    This will add you in db, but yet you have only the default scopes i.e. `rating:read` and `rating:write` which you can check with your jwt. On UI, in right corner there is an option to `Copy Hub Token` and paste in any jwt decoder. for eg. https://jwt.io/
```

Comment:
why do we need to paste in the JWT decoder?

Reply:
just for fun :D....no reason so that user can check scopes